box-ui/src/components/ts/Box/BoxMeeting.tsx

```diff
@@ -12,37 +12,84 @@ const BoxMeeting: FunctionComponent = () => {
         roomName: state && state.roomName ? state.roomName : 'dty',
         domain: state && state.domain ? state.domain : 'meeting.education',
     });
-    const meetingOptions = {
-        userInfo: {
-            email: '',
-            displayName: 'Jitsi-Box',
-        },
-        configOverwrite: {
-            toolbarButtons: [
-                'microphone',
-                'camera',
-                'videoquality',
-                'fodeviceselection',
-                'raisehand',
-                'tileview',
-                'hangup',
-            ],
-            toolbarConfig: {
-                alwaysVisible: true,
+    const meetingOptions = useMemo(
+        () => ({
+            userInfo: {
+                email: '',
+                displayName: 'Jitsi-Box',
             },
-            prejoinConfig: {
-                enabled: false,
+            configOverwrite: {
+                toolbarButtons: [
+                    'microphone',
+                    'camera',
+                    'videoquality',
+                    'fodeviceselection',
+                    'raisehand',
+                    'tileview',
+                    'hangup',
+                ],
+                toolbarConfig: {
+                    alwaysVisible: true,
+                },
+                prejoinConfig: {
+                    enabled: false,
+                },
+                startWithVideoMuted: false,
+                startWithAudioMuted: true,
             },
-            startWithVideoMuted: false,
-            startWithAudioMuted: true,
-        },
-    };
+        }),
+        [],
+    );
+    const [raised, setRaised] = useState(false);
+    const [counter, setCounter] = useState(0);
+    const TimeOutRef = useRef<NodeJS.Timeout>();
+
+    function lowerHand() {
+        setRaised(false);
+        setCounter(0);
+    }
+
+    const updateCounter = useCallback((timeRaised: number) => {
+        if (timeRaised > 0) {
+            setCounter((counter) => counter + 1);
+        }
+        if (timeRaised == 0) {
+            setCounter((counter) => Math.max(0, counter - 1));
+        }
+    }, []);
+
+    function switchHand() {
+        let stop = false;
+        if (counter > 0) {
+            stop = true;
+        }
+        if (stop) {
+            setRaised(true);
+        } else {
+            setRaised(false);
+        }
```

Comment:
```suggestion
            setRaised((counter > 0));
```
it seems that stop = counter > 0